src/components/product.js

```diff
@@ -649,7 +650,7 @@ export default class Product extends Component {
       this.props.closeModal();
       this._userEvent('addVariantToCart');
       this.props.tracker.trackMethod(this.cart.addVariantToCart.bind(this), 'Update Cart', this.selectedVariantTrackingInfo)(this.selectedVariant, this.selectedQuantity);
-      if (this.iframe) {
+      if (!this.modalProduct) {
```

Comment:
`this.iframe` is always undefined since there is no iframe property on the product component, so `setActiveEl` was never called. I also don't see a reason to skip focus tracking in non-iframe mode, so I think it's safe to remove the check